x-pack/plugin/sql/qa/multi-node/src/test/java/org/elasticsearch/xpack/sql/qa/multi_node/RestSqlMultinodeIT.java

```diff
@@ -110,7 +111,7 @@ private void assertCount(RestClient client, int count) throws IOException {
         expected.put("rows", singletonList(singletonList(count)));
 
         Request request = new Request("POST", SQL_QUERY_REST_ENDPOINT);
-        request.setJsonEntity("{\"query\": \"SELECT COUNT(*) FROM test\"" + mode(mode) + "}");
+        request.setJsonEntity("{\"query\": \"SELECT COUNT(*) FROM test\"" + mode(mode) + version(mode) + "}");
```

Comment:
It's worth adding an utility (or two) to handle the query wrapping:
1. one to add mode() and version at the end as parameters
2. another to wrap the given string in a query with the `{\"query\"`, escaped quotes and everything else.

Reply:
That's true. And the fact that the queries are "manually" build can lead to more work whenever the fields list is updated. I've had a stab at mimicking SqlQueryRequestBuilder's interface at building requests and apply the change through the tests [here](https://github.com/bpintea/elasticsearch/commit/392dfc275d00dc420c2907e2405f88cbfeac4b12), but the change is a bit larger, and it might be worth having a separate PR?